### PR TITLE
crush: re-organize the help to clarify ordering

### PR DESCRIPTION
**Pull request by dachary**
The options are grouped according to the stage they relate to

   - input/build
   - tunables adjustments
   - modifications
   - display/test
   - output

Signed-off-by: Loic Dachary <ldachary@redhat.com>

### CODE REVIEW

File: src/test/cli/crushtool/help.t [hunk 1]
```diff
@@ -1,23 +1,48 @@
...
+   - tunables adjustments
+   - modifications
+   - display/test
+   - output
+  
+  Options that are not specific to a stage.
+  
+     [--infn|-i infile]
+                           read the crush map from infile
+
```
Comment (liewegas) [inline src/test/cli/crushtool/help.t]: wouldn't this go in the input/build stage?

**Comment by liewegas** [comment]
-i quibble aside, :+1: !

**Comment by dachary** [comment]
@liewegas right, it turns out that there are no options common to all, that simplifies things. repushed !

**Comment by dachary** [comment]
@liewegas rebased & repushed

**Pull request merged**